src/components/Carousel/Carousel.js

```diff
@@ -10,23 +11,22 @@ const Carousel = ({ items }) => {
     setCurrentItem(currentItem == items.length - 1 ? 0 : currentItem + 1);
   };
 
+  const getData = () => {
+    return {
+      name: items[currentItem].data.name,
+      url: items[currentItem].data.main_image.url,
+      alt: items[currentItem].data.main_image.alt,
+      classSection: "carousel_main",
+    };
+  };
+
   return (
     <>
-      <h1>Carousel Section</h1>
-      <article>
-        <section className="carousel_main">
-          <h3>{items[currentItem].data.name}</h3>
-          <img
-            src={items[currentItem].data.main_image.url}
-            alt={items[currentItem].data.main_image.alt}
-          />
-        </section>
-
-        <section className="carousel_controls">
-          <button onClick={handledClickPrevious}>Previous</button>
-          <button onClick={handledClickNext}>Next</button>
-        </section>
-      </article>
+      <Card data={getData()} />
```

Comment:
For passing data to a component, there's no need to wrap it within a function that returns an object. You can pass each value as a prop. If any of the values changes, then the component will be updated.